src/lib.rs

```diff
@@ -276,6 +276,9 @@
 #![cfg_attr(rustbuild, feature(staged_api, rustc_private))]
 #![cfg_attr(rustbuild, unstable(feature = "rustc_private", issue = "27812"))]
 
+#![feature(is_sorted)]
+#![feature(const_type_id)]
```

Comment:
I think we just stabilise these const methods on-demand. I can’t see why this one shouldn’t be stabilised, but will look into it.

Reply:
See: https://github.com/rust-lang/rust/pull/72488